docs/pages/1 - Intro to Mill.md

```diff
@@ -701,13 +701,12 @@ mill mill.scalalib.Dependency/showUpdates
 mill mill.scalalib.Dependency/showUpdates --allowPreRelease true # also show pre-release versions
 ```
 
-## IntelliJ Support
+## IDE Support
 
-Mill supports IntelliJ by default. Use `mill mill.scalalib.GenIdea/idea` to
-generate an IntelliJ project config for your build.
+Mill supports any IDE that is compatible with [BSP](https://build-server-protocol.github.io/), sush as IntelliJ, VSCode
+or Emacs. Use `mill mill.BSP/install` to generate the BSP project config for your build.
 
-This also configures IntelliJ to allow easy navigate & code-completion within
-your build file itself.
+This also configures your IDE to allow easy navigate & code-completion within your build file itself.
```

Comment:
I'd rather not replace the IntelliJ support documentation, but add the one for BSP before it. I work on various larger projects that's still struggle with BSP, but work fine with direct IJ support. Feel free to add a note that makes clear, that the BSP support in the upcoming preferred way and IJ is meant as a legacy / fallback.

Reply:
Corrected